docs/6-rendering-react.md

```diff
@@ -15,19 +17,60 @@ export default function Article({ opti }: Props) {
   return (
     <main>
       <h1>{opti.heading}</h1>
-      <div dangerouslySetInnerHTML={{ __html: opti.body?.html ?? '' }} />
+      <RichText content={opti.body?.json} />
     </main>
   );
 }
 ```
 
-> [!WARNING]
-> Using [React `dangerouslySetInnerHTML`](https://legacy.reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) can introduce XSS risks. In production environments you should sanitize the HTML or use `opti.body?.json` (which gives the content as JSON)
+> [!TIP]
+> Using the `<RichText/>` component is the recommended way to render rich text content. It's safer than `dangerouslySetInnerHTML` as it doesn't rely on HTML parsing, and allows you to customize how elements are rendered with your own React components.
+
+### Customizing Rich Text Elements
```

Comment:
We need to consider re-organizing the documentation, because right now we only have a "getting started tutorial".

My feeling is that this part is a bit too advanced for that.

Don't delete the documentation because it's very valuable :), we just need to think where to move it!